vscode/src/user_interface/Viewers.ts

```diff
@@ -65,6 +68,31 @@ export class CodiosDataProvider implements vscode.TreeDataProvider<vscode.TreeIt
       }
     }
   }
+
+  /**
+   * Format given milliseconds to ISO 8601 time format.
+   * @param ms Milliseconds to format.
+   */
+  private getTimeDescription(ms: number): string {
+    const roundedSec = Math.round(+('0.' + (ms % 1000))) * 1000;
+    const srt = parser.msToSrt(ms + roundedSec);
```

Comment:
Same comment regarding the dependency in the previous PR

Reply:
Let me know your thoughts on my reply in the previous PR.